cmd/up.go

```diff
@@ -424,6 +432,11 @@ func updateShipment(username string, token string, currentShipment *ShipmentEnvi
 
 	} //envvars
 
+	//if healthcheck settings are specified in yaml, update them
+	if shipment.HealthcheckTimeoutSeconds != nil {
```

Comment:
I feel like this check should be in the `updatePortHealthcheckSettings` function. Also are we missing a check for interval?

Reply:
Good catch.  I'll update.